### PR TITLE
add additional disk space support

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

Add additional disk space support 
Resolves: #821.

### CODE REVIEW

**Review by Serpentiel — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ nav_order: 1
 
 # Changelog
 
+## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
+
+- Add additional disc space support
```
Comment (Serpentiel) [inline CHANGELOG.md:12]: please change all `disc` to `disk` everywhere, see: https://en.wikipedia.org/wiki/Spelling_of_disc

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ivan-savciuc) [reply CHANGELOG.md]: done :+1:

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**